nifi-mock/src/test/java/org/apache/nifi/util/validator/InstrumentedValidator.java

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
This one should go in the main sources so it's available to other modules to test their validators, otherwise it is only available to test classes in this module

Reply:
ok, working on it